lib/Doctrine/AuditLog.php

```diff
@@ -128,6 +128,7 @@ public function getVersion(Doctrine_Record $record, $version, $hydrationMode = D
             ->createQuery();
 
         $values = array();
+        $conditions = array();
```

Comment:
All 3 changes in this file were because an array was being appended to without actually being defined beforehand (and then being returned or used outside of the scope that the appending was being done).